backend/mailman_lambda.py

```diff
@@ -27,7 +28,9 @@ def lambda_handler(event, context):
             )
             item = response['Item']
             oauth_token = item['dropbox_oauth_token']['S']
-            dropbox_client.upload_file_to_dropbox(email.text, '/Mailman', email.subject + '.txt', oauth_token)
+            formatted_email = FormattedEmail(email)
+            path = '/Mailman/%s' % formatted_email.foldername
```

Comment:
Wouldn't this just set it to the subject not with an extension?

Reply:
@rogebrd Yeah, we do this because we need the base path to build the attachment file paths.